notebooks/CubevizExample.ipynb

```diff
@@ -38,6 +38,40 @@
     "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
     "viz.load_data(fn)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Alternate ways to show the Cubeviz UI\n",
+    "\n",
+    "The above shows the Cubeviz UI inline with the notebook.  If you are in jupyter lab you can instead have the Cubeviz UI appear in a separate tab or as a \"sidecar\" (a split UI pane to the side of the notebook).  The below should do this (but only in jupyterlab.  If you are in the \"classic\" notebook it will show the same thing as `viz.app`)"
```

Comment:
```suggestion
    "The above shows the Cubeviz UI inline with the notebook.  If you are in JupyterLab you can instead have the Cubeviz UI appear in a separate tab or as a \"sidecar\" (a split UI pane to the side of the notebook).  The below should do this (but only in JupyterLab.  If you are in the \"classic\" notebook it will show the same thing as `viz.app`)"
```